.gitmodules

```diff
@@ -12,8 +12,8 @@
     branch = main
 [submodule "external/Java.Interop"]
     path = external/Java.Interop
-    url = https://github.com/xamarin/java.interop.git
-    branch = main
+    url = https://github.com/jonpryor/java.interop.git
```

Comment:
These changes clearly need to be reverted before we can merge, and https://github.com/xamarin/java.interop/pull/1007 needs to be merged, but *otherwise*…